site/blog_collections/_running/2026-01-25-2025_running_year_in_review.md

```diff
@@ -0,0 +1,91 @@
+---
+layout: post
+title: "2025 Running Year in Review"
+description: "A summary of my running in 2025"
+date: 2026-01-25 17:00:00 +0100
+comments: true
+tags: running ultrarunning utmb review
+draft: true
+---
+
+This is a summary of how 2025 turned out to be for me in terms of running. I have kept it very short when it comes to mentioning the races etc., otherwise it would be a neverending story 😉.
+
+# Training Volume
+According to Strava, in 2025 I ran `4850K` over `464h20m` with a total elevation gain landing on `36.5K`. That's equivalent to an average of `93K` / `8.9h` per week.
+
+{% capture table_content %}
+|------------|------------|
+| <img src="/img_running/2026-01-25/strava1.jpg" data-src="/img_running/2026-01-25/strava1.jpg" alt="" class="spotlight w-100 pl-2 pr-2" style="max-width: 350px" /> | <img src="/img_running/2026-01-25/strava2.jpg" data-src="/img_running/2026-01-25/strava2.jpg" alt="" class="spotlight w-100 pl-2 pr-2" style="max-width: 350px" /> |
+{% endcapture %}
+{{ table_content | markdownify }}
+
+The volume is slightly less than 2024 (5000K / 501h) - primarily due to Tuesday cross training in the first half of the year (stair stepper, prep for Mozart).
+
+# Running Moments
+
+## Achievements During the Year
+- 3 half marathons at Strava sub-90 minutes. But all UNOFFICIAL as all the race times was above 😆😭
+  - Cross Island Bornholm. Official: 1h30m10s. Strava half: 1h29m39s
+  - Odense City Half. Official: 1h31m. Strava half: 1h29m46s
+  - Bestseller Aarhus City Half. Official: 1h30m37s. Strava half: 1h29m50s
+- ITRA Advanced 4 level achieved
+
+## December
+- Strava PBs:
+  - 10K PB during training at 39m50s (during which also: 5K 2nd fastest PB at 19m45s)
+- Highest ITRA Score ever
```

Comment:
I want you to NOT group achievements by month. But rather by kind (time/distance) - like for the other 2024 report. Study that and use the same approach here.

Make the achievement short and sweet. For example a table with 4 columns: kind (1K, 21K etc), pace (min-sec per k), time (this is the new PB), comment (where/how: training vs race). Again with a link to any race if applicable.